pegasus/migrations/094_create_forms_geos_table.rb

```diff
@@ -0,0 +1,17 @@
+Sequel.migration do
+  change do
+    create_table(:forms_geos, charset: 'utf8') do
+      primary_key :id, unsigned: true, null: false
+      foreign_key :forms_id, null: false, index: true
+      DateTime :created_at, null: false
+      DateTime :updated_at, null: false
```

Comment:
No. Reflecting more, I think I'm going to remove the `indexed_at` column on the dashboard side.